sphinx/builders/html.py

```diff
@@ -1389,6 +1389,7 @@ def setup(app):
     app.add_config_value('html_search_scorer', '', None)
     app.add_config_value('html_scaled_image_link', True, 'html')
     app.add_config_value('html_experimental_html5_writer', None, 'html')
+    app.add_config_value('html_footnotes_options', '', 'html')
```

Comment:
Please add fourth argument using `sphinx.config.ENUM` like following:
```
    app.add_config_value('latex_engine', default_latex_engine, None,
                         ENUM('pdflatex', 'xelatex', 'lualatex', 'platex'))
```
It detects wrong values on running..

Reply:
i don't think that `ENUM` is feasible to validate the values as multiple values can be set as flags. e.g. these are valid: `'bottom,tooltip'`, `['bottom', 'rubric']`.